doc/test-framework/writing-e2e-tests.md

```diff
@@ -250,9 +250,9 @@ $ operator-sdk test local ./test/e2e --namespace operator-test
 
 To run the operator itself locally during the tests instead of starting a deployment in the cluster, you can use the
 `--up-local` flag. This mode will still create global resources, but by default will not create any in-cluster namespaced
-resources unless the user specifies one through the `--namespaced-manifest` flag. (Note: the `--up-local` flag requires
-the `--namespace` flag:
-**NOTE**: The command will NOT create the `--namespace`.  Then, be sure that you are specifying a valid namespace. 
+resources unless the user specifies one through the `--namespaced-manifest` flag. (**NOTE**: The `--up-local` flag requires
+the `--namespace` flag,
+and the command will NOT create the `namespace`, then, be sure that you are specifying a valid namespace). 
```

Comment:
I think is too long the sentence, also IMHO it would be better bellow the phase as a **NOTE** in the doc. WDYT could we change it?

Reply:
`Note: the --up-local flag requires the --namespace flag: NOTE: The command will NOT create the --namespace. Then, be sure that you are specifying a valid namespace.` 
Are you suggesting something like this?
Wouldn't it be the note followed by note extends first note then?

Reply:
I am suggesting:
- Do not perform changes in the current explanation
- Then, just add a **NOTE:** in the next line and not in the same paragraph. 

WDYT?

Reply:
`.....(Note: the --up-local flag requires the --namespace flag).`

`NOTE: The command will NOT create the namespace. Then, be sure that you are specifying a valid namespace.`

Something like this!
This is doable, and yeah more elegant.

Reply:
It is in the same line which is not the docs standards. The note in the same line with () makes harder the understanding. 

See the following example. 

https://github.com/operator-framework/operator-sdk/blob/491cdf6d9f600514963f5f29ccff34a5bfe9ba4e/doc/user/unit-testing.md#L155

Reply:
Ok, so I should remove the existing note from the paragraph and add a new note to the next line but not in the same paragraph.
`paragraph...`

`**NOTE**: The --up-local flag requires the --namespace flag and the command will NOT create the --namespace. Then, be sure that you are specifying a valid namespace`

Something like this! This was your code suggestion!

Reply:
Committed changes have a look if its OK or not?